src/middlewared/middlewared/plugins/reporting.py

```diff
@@ -666,6 +666,26 @@ class NFSStatPlugin(RRDBase):
     )
 
 
+class UPSBatteryChargePlugin(RRDBase):
+
+    plugin = 'nut-ups'
+    title = 'UPS Battery Statistics'
+    vertical_label = 'Percentage'
+    rrd_types = (
+        ('percent-charge', 'value', None),
+    )
+
+
+class UPSRemainingBatteryPlugin(RRDBase):
+
+    plugin = 'nut-ups'
+    title = 'UPS Remaining Battery Statistics'
```

Comment:
```suggestion
    title = 'UPS Battery Time Remaining Statistics'
```
Do we need the word "Statistics"?

Reply:
I think we should as that is what they depict. It's not just a single value but a range of values